CromIAM/src/main/scala/cromiam/webservice/WomtoolRouteSupport.scala

```diff
@@ -0,0 +1,23 @@
+package cromiam.webservice
+
+import akka.http.scaladsl.server.Directives._
+import cromiam.cromwell.CromwellClient
+import cromwell.api.model._
+
+trait WomtoolRouteSupport extends RequestSupport {
+  // When this trait is mixed into `CromIamApiService` the value of `cromwellClient` is the reader (non-abort) address
+  val cromwellClient: CromwellClient
+
+  val womtoolRoutes =
+    path("api" / "womtool" / Segment / "describe") { _ =>
+      post {
+        extractStrictRequest { req =>
+          complete {
+            // This endpoint requires authn which it gets for free from the proxy, does not care about authz
```

Comment:
Regardless of how many thumbs I have I am not going to merge this until I have personally received confirmation of this from @adrazhi

Reply:
Proxy writeup: https://medium.com/@lukwam/open-sourcing-openidc-proxy-67cc3870f168

Reply:
Confirmed!